src/natcap/invest/habitat_quality.py

```diff
@@ -298,17 +298,6 @@
                 "filtered_[THREAT]_aligned.tif": {
                     "about": "Filtered threat raster",
                     "bands": {1: {"type": "ratio"}},
-                },
-                "kernels": {
```

Comment:
This was not actually used anywhere.

Reply:
Ah, this looks like it's left over from the old convolution-based workflow we recently reverted.  Thanks for catching this!